thirdparty/README.md

```diff
@@ -890,6 +891,9 @@ Files extracted from upstream source:
 See `thorvg/update-thorvg.sh` for extraction instructions. Set the version
 number and run the script.
 
+The `tvgRender.h` file was modified to change a `==` operator to const,
+allowing it to function as expected in C++20.
```

Comment:
This would be worth sending upstream, which is pretty reactive.
So by the time we're ready to merge this PR the upstream thorvg should have the fix already.

Reply:
Part of upstream as of [`v0.14.1`](https://github.com/thorvg/thorvg/releases/tag/v0.14.1)!